apps/api/src/service/event.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Event, PrismaClient } from '@prisma/client';
+import { EventRepository } from '../repository';
+
+export class EventService {
+  private readonly eventRepository: EventRepository;
+  constructor(prisma: PrismaClient) {
+    this.eventRepository = new EventRepository(prisma);
+  }
+
+  async createEvent(event: Event): Promise<Event> {
+    return await this.eventRepository.create(event);
+  }
+
+  async getEvents(): Promise<Event[]> {
+    return this.eventRepository.findMany();
+  }
+
+  async getEventById(id: string): Promise<Event> {
+    return this.eventRepository.findOne(id);
+  }
+
+  async updateEvent(id: string, event: Event): Promise<Event> {
+    return await this.eventRepository.update(id, event);
+  }
+
+  async endEvent(id: string): Promise<Event> {
+    return await this.eventRepository.update(id, { endAt: new Date() });
+  }
+  async addAthleteToEvent(eventId: string, athleteId: string): Promise<Event> {
+    return await this.eventRepository.addAthlete(eventId, athleteId);
+  }
+  async removeAthleteFromEvent(
+    eventId: string,
+    athleteId: string
+  ): Promise<Event> {
+    return await this.eventRepository.removeAthlete(eventId, athleteId);
+  }
```

Comment:
Juste un poil d'air stp :)
```suggestion
  async endEvent(id: string): Promise<Event> {
    return await this.eventRepository.update(id, { endAt: new Date() });
  }
  
  async addAthleteToEvent(eventId: string, athleteId: string): Promise<Event> {
    return await this.eventRepository.addAthlete(eventId, athleteId);
  }
  
  async removeAthleteFromEvent(
    eventId: string,
    athleteId: string
  ): Promise<Event> {
    return await this.eventRepository.removeAthlete(eventId, athleteId);
  }
```